src/material-experimental/mdc-slider/slider.spec.ts

```diff
@@ -6,14 +6,325 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
+import {dispatchMouseEvent, dispatchPointerEvent} from '@angular/cdk/testing/private';
+import {Component, DebugElement, Type} from '@angular/core';
+import {ComponentFixture, TestBed, waitForAsync} from '@angular/core/testing';
+import {RippleRef, RippleState} from '@angular/material/core';
+import {By} from '@angular/platform-browser';
+import {Thumb} from '@material/slider';
+import {MatSliderModule} from './module';
+import {MatSlider, MatSliderThumb, MatSliderVisualThumb} from './slider';
 
-/* tslint:disable-next-line:no-unused-variable */
-import {MatSlider} from './index';
+describe('MDC-based MatSlider' , () => {
+  beforeAll(() => {
+    // Mock #setPointerCapture as it throws errors on pointerdown without a real pointerId.
+    spyOn(Element.prototype, 'setPointerCapture');
```

Comment:
As far as I'm aware, `spyOn` gets cleaned up after each test. I wonder whether having this inside `beforeAll` isn't going to cause issues down the line.

Reply:
That can't be the case as `setPointerCapture` already gets called by a bunch of the unit tests